src/Snakemake/rules/VCF_fix/Multibp_SNV.smk

```diff
@@ -4,8 +4,19 @@ rule Find_multibp_SNV:
         vcf="recall/{sample}.ensemble.vep.exon.soft_filter.vcf",
         ref=config["reference"]["ref"],
     output:
-        vcf="recall/{sample}.ensemble.vep.exon.soft_filter.multibp.vcf",
+        vcf=temp("recall/{sample}.ensemble.vep.exon.soft_filter.multibp.vcf.temp"),
     singularity:
-        config["singularity"]["python"]
+        config["singularity"].get("python_samtools", config["singularity"].get("default", ""))
     script:
         "../../../scripts/python/Multibp_SNV.py"
+
+
+rule sort_multiplebp_vcf:
+    input:
+        vcf="recall/{sample}.ensemble.vep.exon.soft_filter.multibp.vcf.temp",
+    output:
+        vcf="recall/{sample}.ensemble.vep.exon.soft_filter.multibp.vcf",
+    singularity:
+        config["singularity"].get("bcftools", config["singularity"].get("default", ""))
+    shell:
+        "bcftools sort -o {output.vcf} -O v {input.vcf}"
```

Comment:
Maybe use a [wrapper](https://snakemake-wrappers.readthedocs.io/en/0.68.0/wrappers/bcftools/sort.html)? With it you can easily change output format

Reply:
Added a wrapper